### PR TITLE
Add badges to readme 

### PR DESCRIPTION
**Pull request by pwnb0y**
#49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # rhof
 
+[![Build](https://github.com/pwnb0y/rhof/actions/workflows/nodejs.yml/badge.svg)](https://github.com/pwnb0y/rhof/actions/workflows/nodejs.yml)
```
Comment (arshadkazmi42) [inline README.md:3]: `pwnb0y` => `gh-conf`

seems there is some issue with the `npm-badger` tool. It used your username.

Can you update this? Find and replace in the whole file

File: README.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+[![NPM Version](https://img.shields.io/npm/v/rhof.svg)](https://www.npmjs.com/package/rhof)
+[![NPM Downloads](https://img.shields.io/npm/dt/rhof.svg)](https://www.npmjs.com/package/rhof)
+[![Github Repo Size](https://img.shields.io/github/repo-size/pwnb0y/rhof.svg)](https://github.com/pwnb0y/rhof)
+[![LICENSE](https://img.shields.io/npm/l/rhof.svg)](https://github.com/pwnb0y/rhof/blob/master/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/pwnb0y/rhof.svg)](https://github.com/pwnb0y/rhof/graphs/contributors)
+[![Commit](https://img.shields.io/github/last-commit/pwnb0y/rhof.svg)](https://github.com/pwnb0y/rhof/commits/master)
+
+
 <!-- Add Badges here -->
```
Comment (arshadkazmi42) [inline README.md:12]: Can we also remove these from line 3 to 9 since we are using badges added by npm-badger now

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

I have added two minor changes. I forgot to mention this in the issue, sorry about that.

Can you make these changes? We can get this merged once the changes are done

File: README.md (hunk 1 above)
Comment (arshadkazmi42) [reply README.md]: @pwnb0y I think my comments were not clear.

I meant in these lines https://github.com/gh-conf/rhof/pull/50/commits/916dc6db699a2321cc12141443e5aad7d223208f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3-R9

replace `pwnb0y` => `gh-conf`

File: README.md (hunk 2 above)
Comment (arshadkazmi42) [reply README.md]: Remove these lines https://github.com/gh-conf/rhof/pull/50/commits/916dc6db699a2321cc12141443e5aad7d223208f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R12-R18

**Comment by arshadkazmi42** [comment]
@pwnb0y I think my comments were not very clear. I updated comments we will need changes according to that. Seems latest commit reverted all the changes and now there is no changes are present in the PR

**Comment by pwnb0y** [comment]
Hey @arshadkazmi42 can you review it again

File: README.md (hunk 2 above)
Comment (arshadkazmi42) [reply README.md:12]: @pwnb0y this is still pending. We can remove line 12 - 18 in README.md

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
You have missed one

**Comment by pwnb0y** [comment]
Ok

**Review by arshadkazmi42 — APPROVED** [review]
LGTM 🎉

**Pull request merged**